common/source/docs/common-flywoo-f745.rst

```diff
@@ -70,15 +91,30 @@ UART3 supports RX and TX DMA. UART1, UART2, UART4, and UART6 supports TX DMA. UA
 RC Input
 ========
 
-RC input is configured by default via the USART3 RX input. It supports all RC protocols except PPM. FPort and full duplex protocols will use both RX6 and TX6. The 4.5V output pin is powered when the USB is connected to allow setup of receiver without connecting to the main battery.
+RC input is configured on the RX3 (UART3_RX) pin. It supports all RC protocols except PPM. See :ref:`common-rc-systems` for details for a specific RC system. :ref:`SERIAL3_PROTOCOL<SERIAL3_PROTOCOL>` is set to "23", by default, to enable this.
+
+- SBUS/DSM/SRXL connects to the RX3 pin but SBUS requires that the :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` be set to "3".
 
-.. note:: If the receiver is SBUS, then :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` must be set to "1" to provide inversion for detection. If the receiver is FPort, then the receiver must be tied to the USART3 TX pin and :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` = 7 (invert TX/RX, half duplex) and :ref:`RSSI_TYPE<RSSI_TYPE>` =3.
+- FPort requires connection to TX3 and :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` be set to "7".
 
+- CRSF also requires a TX3 connection, in addition to RX3, and automatically provides telemetry. Set :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` to "0".
 
-Dshot Capability
-================
+- SRXL2 requires a connection to TX6 and automatically provides telemetry.  Set :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` to "4".
 
-All motor 1-4 outputs are bi-directional Dshot and PWM capable. However, mixing Dshot and normal PWM operation for outputs is restricted into groups, ie. enabling Dshot for an output in a group requires that ALL outputs in that group be configured and used as Dshot, rather than PWM outputs. The output groups that must be the same (PWM rate or Dshot, when configured as a normal servo/motor output) are: 1/2, 3/4/10, 5, 6, 7/8, 9.
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+PWM Output
+==========
+
+The GOKU GN 745 AIO supports up to 8 PWM outputs. The pads for motor output ESC1 to ESC4 on the above diagram are the first 4 outputs, there are four additional pads for PWM 5-8. All 8 outputs support DShot as well as all PWM types.
+
+The Nano version has 4 esc signals, current and voltage sense inputs on an external connector. The remaining outputs are on solder pads.
+
+The PWM are in 5 groups: 1/2/7/8, 3/4, 5, 6
+
+Channels within the same group need to use the same output rate. If
+any channel in a group uses DShot then all channels in the group need
+to use DShot. PWM 1-4 support bidirectional dshot in the AIO version, only PWM2-4 support bidirectional dshot in the Nano version.
```

Comment:
I don't think this is right - should be 1-4 on both boards

Reply:
why is the hwdef different then for the AIO and Nano....thought the second I2C needed a DMA and it was taken from PWM1....but I am not sure I understand how the hwdef for BIDIR tags work

Reply:
Just different resource allocations. I would have had to move something around to make it work, but I just checked - bi-dir on PWM 1-4 on the nano

Reply:
Will change